rpc/ub/plot.sh

```diff
@@ -201,7 +201,7 @@ nr_conns=96,nr_msgs=100  msg_len  64 128 256 512
 EOF
 } | while read -a ARGS; do
     CSV=$((++i)).csv
-    gen_csv $CSV $TMP ${ARGS[@]}
-    gen_script ${ARGS[0]} $CSV ${ARGS[1]} >$TMP
-    gnuplot $TMP
+    gen_csv $CSV "$TMP" "${ARGS[@]}"
```

Comment:
This does not appear to be correct. Enclosing '${ARGS[@]}' in double quotes will concatenate all the arguments in a single string.

Reply:
Changed as per comment.